lib/mayaUsd/render/mayaToHydra/renderOverride.cpp

```diff
@@ -264,6 +264,12 @@ MtohRenderOverride::MtohRenderOverride(const MtohRendererDescription& desc)
 
 MtohRenderOverride::~MtohRenderOverride()
 {
+    const Ufe::GlobalSelection::Ptr& ufeSelection = Ufe::GlobalSelection::get();
```

Comment:
You will need to protect this change with the `WANT_UFE_BUILD` flag - see what we do when we register the observer:
https://github.com/Autodesk/maya-usd/blob/b733ccf51d7942ee73ecfab2a1b1ac68de4076a0/lib/mayaUsd/render/mayaToHydra/renderOverride.cpp#L256